src/Xamarin.Android.Build.Tasks/Utilities/NativeAssemblyGenerator/SymbolType.cs

```diff
@@ -0,0 +1,13 @@
+namespace Xamarin.Android.Tasks
+{
+	enum SymbolType
+	{
+		Function,
+		GnuIndirectFunction,
```

Comment:
Why include enum members that aren't really *used* anywhere? `WriteSymbolType()` "supports" it, but I don't see anywhere that uses it.  Why have this member at all?

Reply:
For completeness. We don't generate executable code but I didn't see harm in naming all the possible symbol (and section) types, if only to show that the code is backed by a thorough understanding of the topic.